src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetaDataTest.java

```diff
@@ -887,6 +920,84 @@ public void testGetImportedKeysDW() throws SQLException {
+            getColumnMetaDataClass.put(DECIMAL_DIGITS, Integer.class);
+            getColumnMetaDataClass.put(NUM_PREC_RADIX, Integer.class);
+            getColumnMetaDataClass.put(NULLABLE, Integer.class);
+            getColumnMetaDataClass.put(REMARKS, String.class);
+            getColumnMetaDataClass.put(COLUMN_DEF, String.class);
+            getColumnMetaDataClass.put(SQL_DATA_TYPE, Integer.class);
+            getColumnMetaDataClass.put(SQL_DATETIME_SUB, Integer.class);
+            getColumnMetaDataClass.put(CHAR_OCTET_LENGTH, Integer.class);
+            getColumnMetaDataClass.put(ORDINAL_POSITION, Integer.class);
+            getColumnMetaDataClass.put(IS_NULLABLE, String.class);
+            getColumnMetaDataClass.put(SCOPE_CATALOG, String.class);
+            getColumnMetaDataClass.put(SCOPE_SCHEMA, String.class);
+            getColumnMetaDataClass.put(SCOPE_TABLE, String.class);
+            getColumnMetaDataClass.put(SOURCE_DATA_TYPE, Short.class);
+            getColumnMetaDataClass.put(IS_AUTOINCREMENT, String.class);
+            getColumnMetaDataClass.put(IS_GENERATEDCOLUMN, String.class);
+            getColumnMetaDataClass.put(SS_IS_SPARSE, Short.class);
+            getColumnMetaDataClass.put(SS_IS_COLUMN_SET, Short.class);
+            getColumnMetaDataClass.put(SS_UDT_CATALOG_NAME, String.class);
+            getColumnMetaDataClass.put(SS_UDT_SCHEMA_NAME, String.class);
+            getColumnMetaDataClass.put(SS_UDT_ASSEMBLY_TYPE_NAME, String.class);
+            getColumnMetaDataClass.put(SS_XML_SCHEMACOLLECTION_CATALOG_NAME, String.class);
+            getColumnMetaDataClass.put(SS_XML_SCHEMACOLLECTION_SCHEMA_NAME, String.class);
+            getColumnMetaDataClass.put(SS_XML_SCHEMACOLLECTION_NAME, String.class);
+    	}
+    	
+        try (Connection conn = getConnection()) {
+            ResultSetMetaData metadata = conn.getMetaData().getColumns(null, null, tableName, null).getMetaData();
+
+            for (int i = 1; i < metadata.getColumnCount(); i++) {
+                // Ensure that there is a data type for every metadata column
+                String columnLabel = metadata.getColumnLabel(i);
+                String columnClassName = metadata.getColumnClassName(i);
+                Class<?> columnClass = null;
+                
+                try 
+                {
+                    columnClass = Class.forName(columnClassName);
+                }
+                catch (ClassNotFoundException ex)
+                {
+                    MessageFormat form = new MessageFormat(
+                                TestResource.getResource("R_classLoaderNotFoundForColumnType"));
+                    Object[] msgArgs = {columnClassName, columnLabel};
+                    fail(form.format(msgArgs));
+                }
+                
+                Class<?> expectedClass = getColumnMetaDataClass.get(columnLabel);
+                                
+                assert(expectedClass != null);
```

Comment:
add null check instead of just assert

Reply:
This assert has been removed and replaced with a null check and will use AssertionUtils.fail() with a descriptive error message